packages/openapi-ts/src/utils/write/services.ts

```diff
@@ -78,6 +78,14 @@ const toOperationComment = (operation: Operation) => {
       (p) =>
         `@param ${p.name} ${p.description ? escapeComment(p.description) : ''}`,
     );
+  } else if (operation.parameters.length) {
```

Comment:
Make this the top level condition and then inside check for useOptions, it's going to be a bit easier to read